### PR TITLE
IDE-602 KEL Syntax Check v Generate

### PR DESCRIPTION
**Pull request by dehilsterlexis**
@GordonSmith for review. I want to wait a week so that Brenton has time to test with with KEL.

### CODE REVIEW

File: en_us/en_us.rc [hunk 1]
```diff
@@ -1997,6 +1997,7 @@ BEGIN
     BEGIN
         MENUITEM "&Submit",                     ID_GO_SUBMIT
         MENUITEM "Su&bmit Selected",            ID_GO_SUBMITSELECTED
+        MENUITEM "Generate",                    ID_GO_GENERATE
```
Comment (GordonSmith) [inline en_us/en_us.rc:2000]: Should add a ALT key option.

**Comment by dehilsterlexis** [comment]
@GordonSmith for review

**Comment by richardkchapman** [comment]
Should this target 7.6.x?

**Comment by GordonSmith** [comment]
It should really target master as its a new feature, but 7.8.x is probably more practice.

**Pull request merged**